### DISCUSSION TITLE
Html positioning

### DISCUSSION THREAD

**Discussion by brianzinn** [brianzinn/react-babylonjs#278] [Q&A] [answered]
hi @dennemark - I'm just noticing that the Html element is not positioned correctly (for me) on the new site:
https://brianzinn.github.io/react-babylonjs/examples/gui/html

I am pretty busy, but can look next week hopefully.  Just wanted to run it by you first in case it was something obvious:
![image](https://user-images.githubusercontent.com/542756/221908729-06582e90-238c-49f2-a4e5-4447a0f8b4e3.png)

I ported from Storybook to Gatsby without expecting a breaking change - hopefully I did that correct.  Cheers.

edit: fixed link

**Comment by dennemark** [comment, marked as answer]
Add a position: relative to the div around canvas :)
![image](https://user-images.githubusercontent.com/29654902/222075292-e2e606fa-4047-4c41-b659-0908626ef7ee.png)

**Reply by brianzinn** [reply]
thanks - that was it!
